solo-io/gloo
# Documentation and Nightly CI Followup

# Description

Minor improvements to our CI pipeline and developer documenation

## CI changes
- Adjusted condition for running peformance tests on 1.15 branch

## Docs changes
- Added a README for `devel/tools`

# Context
- In solo-projects PR, the bug in the nightly performance condition was identified: https://github.com/solo-io/solo-projects/pull/5283#discussion_r1293749882
- In feedback during a team demo, the need for a tools readme was identified

## Testing steps
- Run regression tests locally, using the 1.15 branch and see that the performance tests are run, and the slack notification includes the branch

## Notes for reviewers
-

# Checklist:
